tools/checkpoint_conversion/convert_bert_checkpoints.py

```diff
@@ -0,0 +1,631 @@
+import os
+import tensorflow as tf
+import transformers
+from absl import app
+from absl import flags
+from tensorflow import keras
+import tensorflow_models as tfm
+import keras_nlp
+import json
+
+FLAGS = flags.FLAGS
+
+PRESET_MAP = {
+    "bert_base_cased": {'base':"roberta.base",
```

Comment:
Are these big dicts actually used anywhere? They don't seem to be.

What you probably need is a mapping from our preset names to checkpoint files, and that's it!

Reply:
they are used in the configuration of the model . Where I can find the list of checkpoint files to map them?